src/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do List</title>
+    <script src="https://kit.fontawesome.com/7222a8b277.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <div class="total">
+      <div class="title">
+        <h>Today's To DO</h>
+        <i class="fa-solid fa-arrows-rotate"></i>
+      </div>
+
+      <form class="add-book" id="add-book">
+          <input 
+            type="text"
+            class="newToDo" 
+            name="newToDo" 
+            id="newToDo" 
+            placeholder="Add to your list...">    
+          <button type="button" class="add_button"><i class="fa-solid fa-plus"></i></button>         
+      </form>
+  
+      <div class="addTask"></div>
+      <button type="button" class="remove_button">Clear all completed</button>
+  </div>
+
+</body>
+<script src="./index.js"></script>

```

Comment:
This \<script> line iI believe isn't neccessary when using ES6 ann webpack because the reference to the index.js file should be in the package.json file.